_includes/js/files.md

```diff
@@ -111,9 +111,22 @@ Parse.Cloud.httpRequest({ url: profilePhoto.url() }).then(function(response) {
 });
 ```
 
+
 ## Deleting Files
 
-You can delete files that are referenced by objects using the [REST API]({{ site.baseUrl }}/rest/guide/#deleting-files). You will need to provide the master key in order to be allowed to delete a file.
+Starting with Parse Server 4.2.0, you can delete files using cloud code.
```

Comment:
This sounds like deleting files was only possible since Parse Server 4.2. I suggest rewording that it is clear that prior to 4.2. the REST API can be used.